src/EprPrnIntegration.Common/Service/EmailService.cs

```diff
@@ -98,21 +99,16 @@ public void SendValidationErrorPrnEmail(string csvData, DateTime reportDate)
 
     public void SendIssuedPrnsReconciliationEmailToNpwd(DateTime reportDate, int reportCount, string reportCsv)
     {
+        ArgumentException.ThrowIfNullOrWhiteSpace(reportCsv, nameof(reportCsv));
+
         var templateId = _messagingConfig.NpwdReconcileIssuedPrnsTemplateId;
-        var filename = $"issuedprns_{reportDate:yyyyMMdd}.csv";
         var emailAddress = _messagingConfig.NpwdEmail;
 
         var messagePersonalisation = new Dictionary<string, object>
         {
-            {
-                "report_date", reportDate.ToString("dd/MM/yyyy")
-            },
-            {
-                "report_count", reportCount
-            },
-            {
-                "link_to_file", NotificationClient.PrepareUpload(System.Text.Encoding.UTF8.GetBytes(reportCsv), filename)
-            }
+            ["report_date"] = reportDate.ToString("dd/MM/yyyy"),
+            ["report_count"] = reportCount,
+            ["link_to_file"] = reportCsv
```

Comment:
Since now we are doing changes on Template as well, Can we change link_to_file to something like csvData inline with what we are doing.